internal/mdns/mdns.go

```diff
@@ -9,15 +9,13 @@ import (
 
 	"github.com/jetkvm/kvm/internal/logging"
 	pion_mdns "github.com/pion/mdns/v2"
-	"github.com/rs/zerolog"
 	"golang.org/x/net/ipv4"
 	"golang.org/x/net/ipv6"
 )
 
 type MDNS struct {
 	conn *pion_mdns.Conn
 	lock sync.Mutex
-	l    *zerolog.Logger
```

Comment:
It is NOT legal to hold onto loggers for any duration... so sweeping to eliminate these and always ask from the controlling log.go